packages/jsActions/nanoflow-actions-native/src/client/SignIn.ts

```diff
@@ -32,7 +33,11 @@ export async function SignIn(username?: string, password?: string): Promise<Big>
         const onSuccess = (): void => resolve(new Big(200));
         const onError = (error: { status: number }): void => resolve(new Big(error.status));
 
-        mx.login(username, password, onSuccess, onError as any);
+        if (typeof useAuthToken === "undefined") {
+            mx.login(username, password, onSuccess, onError as any);
+        } else {
+            mx.login2(username, password, Boolean(useAuthToken), onSuccess, onError as any);
```

Comment:
Why do you cast it to `Boolean`? It should already be a boolean right?

Reply:
Yes, that is right.